openapi-v1.yaml

```diff
@@ -882,6 +882,26 @@ definitions:
         items:
           $ref: "#/definitions/ArrayMetadataEntry"
 
+  ConsolidationConfigEntry:
+    type: object
+    description: key/value pair representing a consolidation config entry
+    properties:
+      key:
+        type: string
+      value:
+        type: string
+
+  ConsolidationConfig:
```

Comment:
Should we use just a map instead of a list of key/values?

Reply:
Yeah works well

Reply:
Maybe rename this to just `Config` or `TileDBConfig` we might want to use this other places?

Reply:
👍